docs/pages/eas-update/introduction.mdx

```diff
@@ -18,7 +19,34 @@ All apps that include the `expo-updates` library have the ability to receive upd
 - To start using EAS Update, continue to the [Getting Started](/eas-update/getting-started) guide.
 - For a complete tutorial of using EAS Update with other EAS services, refer to the [EAS Tutorial](/tutorial/eas/introduction/).
 
-> **info** For a look at some of our favorite features of EAS Update, check out the [6 reasons to use EAS Update blog post](https://expo.dev/blog/6-reasons-to-use-eas-update).
+## JS API for update management
+
+<ContentSpotlight
+  alt="Updates JS API"
+  src="/static/images/eas-update/frontpage/critical-update.png"
+/>
+
+The updates [JavaScript API](/versions/unversioned/sdk/updates/) includes a React hook called `useUpdates()`. This hook provides detailed information about the currently running update and any new updates that are available or have been downloaded. In addition, you can view any errors that were encountered during the update process to help you debug any issues while the app is attempting to update.
+
+The API also provides methods such as `checkForUpdateAsync()` and `fetchUpdateAsync()` which allows you to control when your app checks for and downloads updates.
+
+## Insight tracking
+
+<ContentSpotlight
+  alt="Website dashboard deployment insights"
+  src="/static/images/eas-update/frontpage/insights.png"
+/>
+
+You'll get a [deployments dashboard](https://expo.dev/accounts/[account]/projects/[project]/deployments) that helps visualize which updates are being sent to builds. Updates work in concert with [insights](/eas-insights/introduction/) to provide data on the adoption rates of your updates with your users.
+
+## Republishing for walking back mistakes
```

Comment:
```suggestion
## Republish for reverting mistakes
```

I don't think "walking back" is a helpful word to use in this section's title. Folks will not use words like this when using search. I think reverting might be more appropriate word to use here.